BoxContentSDK/BoxContentSDK/Operations/BOXStreamOperation.m

```diff
@@ -144,12 +144,12 @@ - (void)abortWithError:(NSError *)error
 
 #pragma mark - BOXURLSessionTaskDelegate
 
-- (void)sessionTask:(NSURLSessionTask *)sessionTask didFinishWithResponse:(NSURLResponse *)response error:(NSError *)error
+- (void)processIntermediateResponse:(NSURLResponse *)response
 {
-    [super sessionTask:sessionTask didFinishWithResponse:response error:error];
-
-    if (error != nil &&
-        error.code == BOXContentSDKAPIErrorAccepted) {
+    [super processIntermediateResponse:response];
+    
+    if (self.error != nil &&
+        self.error.code == BOXContentSDKAPIErrorAccepted) {
```

Comment:
BOXStreamOperation uses a sessionTask which takes in a completionHandler (and no other interaction needed). It inherits the same createSessionTask method from BOXAPIOperation.

- (NSURLSessionTask *)createSessionTask
{
    __weak BOXAPIOperation *weakSelf = self;
    NSURLSessionTask *sessionTask = [self.session.urlSessionManager createDataTaskWithRequest:self.APIRequest
                                                completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
                                                    [weakSelf finishURLSessionTaskWithData:data response:response error:error];
                                                }];
    return sessionTask;
}

We should call sessionTask:didFinishWithResponse:error: inside sessionTask's completionHandler, so that subclasses like BOXStreamOperation can override.